src/main/kotlin/com/github/devcordde/devcordbot/commands/fun/SourceCommand.kt

```diff
@@ -16,30 +16,29 @@
 
 package com.github.devcordde.devcordbot.commands.`fun`
 
-import com.github.devcordde.devcordbot.command.AbstractCommand
-import com.github.devcordde.devcordbot.command.AbstractSubCommand
-import com.github.devcordde.devcordbot.command.CommandCategory
-import com.github.devcordde.devcordbot.command.CommandPlace
-import com.github.devcordde.devcordbot.command.context.Arguments
+import com.github.devcordde.devcordbot.command.*
 import com.github.devcordde.devcordbot.command.context.Context
 import com.github.devcordde.devcordbot.command.permission.Permission
 import com.github.devcordde.devcordbot.constants.Embeds
-import com.github.devcordde.devcordbot.util.hasSubCommands
+import net.dv8tion.jda.api.requests.restaction.CommandUpdateAction
 
 /**
  * Source command.
  */
-class SourceCommand : AbstractCommand() {
-    override val aliases: List<String> = listOf("source", "skid", "code")
-    override val displayName: String = "source"
+class SourceCommand : AbstractSingleCommand() {
+    override val name: String = "source"
     override val description: String = "Displays the source code of the bot"
-    override val usage: String = "[command]"
     override val permission: Permission = Permission.ANY
     override val category: CommandCategory = CommandCategory.FUN
     override val commandPlace: CommandPlace = CommandPlace.ALL
 
+    override val options: List<CommandUpdateAction.OptionData> = buildOptions {
+        string("command", "Der Name des Commands für den der SourceCode angezeigt werden soll")
```

Comment:
"Der Name des Commands, für den der Sourcecode angezeigt werden soll"

Doesn't really matter tho, I'll do a PR that revises the messages soon

Reply:
There are similar strings which are lacking commas.

Reply:
I am a dev not a german teacher